howto/stream/oob-data.md

```diff
@@ -266,7 +266,7 @@ try {
     ex.printStackTrace();
 }
 ```
-
+<!-- wokeignore:rule=master -->
 For a complete Android example, see the [out_of_band_v2](https://github.com/anbox-cloud/anbox-streaming-sdk/tree/master/examples/android/out_of_band_v2) project.
```

Comment:
Should we instead rename the default branch on that repo to main?

As far as I can tell from https://github.com/github/renaming, this shouldn't break anything.

Reply:
We discussed this in SU.